app/developers/templates/developers/deliver/manage_form.html

```diff
@@ -25,23 +26,7 @@
       {% with errors = get_flashed_messages(with_categories="dependency_order_error") %}
         {% if errors %}
           {% set type, error = errors[0] %}
-          {% set question_link = url_for("developers.deliver.edit_question", grant_id=grant.id, collection_id=collection.id, section_id=section.id, form_id=db_form.id, question_id=error.question_id) %}
-          {% set depends_on_link = url_for("developers.deliver.edit_question", grant_id=grant.id, collection_id=collection.id, section_id=section.id, form_id=db_form.id, question_id=error.depends_on_question_id) %}
-
-
-          {% set banner_html %}
-            <p class="govuk-body govuk-!-font-weight-bold">You cannot move questions above answers they depend on.</p>
-            <p class="govuk-body govuk-!-font-weight-bold">Question:</p>
-            <p class="govuk-body govuk-!-font-weight-bold">
-              <a class="govuk-notification-banner__link" href="{{ question_link }}">{{ error.question_text }}</a>
-            </p>
-            <p class="govuk-body govuk-!-font-weight-bold">Depends on the answer to:</p>
-            <p class="govuk-body govuk-!-font-weight-bold">
-              <a class="govuk-notification-banner__link" href="{{ depends_on_link }}">{{ error.depends_on_question_text }}</a>
-            </p>
-          {% endset %}
-
-          {{ govukNotificationBanner(params={"role": "alert", "titleText": "Warning", "classes": "app-notification-banner--destructive", "html": banner_html}) }}
+          {{ dependency_banner(error, "You cannot move questions above answers they depend on.", grant.id, collection.id, section.id, db_form.id) }}
```

Comment:
🤏  vibe that "You cannot move questions above answers they depend on." could be part of the context, pulled from the DependencyOrderException message, but that comes from my previous vibe around flash message rendering not being tied to templates